example/src/example_layer.cpp

```
@@ -34,31 +39,142 @@ bool TestLayer::OnEvent(moth_ui::Event const& event) {
+    ImGui::Text("Centre     moth_ui layout + animations");
+    ImGui::Text("           Custom UIButton widget");
+    ImGui::Separator();
+    ImGui::Text("S  save screenshot.png");
+    ImGui::Separator();
+    if (ImGui::Button("Save screenshot")) {
+        m_pendingScreenshot = true;
+    }
+    ImGui::End();
+
+    // Recreate the full-screen target whenever the window is resized.
     moth_ui::IntVec2 const currentSize{ GetWidth(), GetHeight() };
```

Clang (14.0.6)

[error] variable 'currentSize' is not initialized

(cppcoreguidelines-init-variables,-warnings-as-errors)